### ISSUE TITLE
Question: Slide on click element?

### ISSUE THREAD

**Issue by guillaumeduhan** [SSENSE/vue-carousel#160]
Hello and thank you for this awesome module,

I got one question. I would like to slide from page 1 to page 2 on click on an input. I found in the documentation, this function: getNextPage(). But when I put: "@click="getNextPage()", it doesn't trigger the sliding.

Another method?

My code:

```
    <carousel id="demo_carousel">
      <slide class="slides_demo">
          <input type="button" @click="getNextPage()" value="Go" />
      </slide>
      <slide class="slides_demo">
          <p>This is next page</p>
      </slide>
    </carousel>
```

And my method:

```
      methods: {
        getNextPage:function() {
          if (this.currentPage < this.pageCount - 1) {
            return this.currentPage + 1;
          }
          return this.loop ? 0 : this.currentPage;
        }
      }
```

Thanks for help :)

**Comment by quinnlangille** [comment]
Hey @guillaumeduhan, I think this is related to #134 in that since 0.6.7 the scrolling features haven't been working as expected. I think someone is working on a fix, so I'll update you here once I test and merge!

**Comment by quinnlangille** [comment]
Any idea if this has been resolved for you yet? The fix went out a while ago with `v0.6.13`, let me know if it's still happening and I'll switch this to a bug report!
